serverless.yml

```diff
@@ -0,0 +1,11 @@
+service: airkeeper
+
+provider:
```

Comment:
We'll probably need a decent memory limit

Reply:
✅

Reply:
I'm also trying to add scheduling on a fixed rate to this file.